frontend/src/components/AppHeader.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components/macro'
+
+export default function AppHeader() {
+
+    return (
+        <Header>
+            <h1>RavensPerch</h1>
+        </Header>)
+}
+
+const Header = styled.header`
+  background: #454A4D;
+  border-style: none none solid none;
+  border-width: 0px 0px 1px 0px;
+  border-color: #C9E1F2;
+  h1 {
+    padding: 8px;
+    margin: 0px;
+    text-align: center;
+    font-size: 1.6em;
+    font-family: 'Times New Roman';
+    color: #C9E1F2;
+    text-align: left;
+  }
```

Comment:
If you plan to use this kind of styling consistently throughout your app, you could consider moving these to your  global styles.

Reply:
Good Point, thank you. I will consider it later in developing process. As the moment its only meant to serve as "dummy styling" to see changes happen in frontend.